client/client.js

```diff
@@ -35,6 +66,11 @@ function Client(apiUrl) {
   // Populates the cache of models so you can access them with
   // client.models.ModelName.
   client.models();
+
+  // Override and implement the remote methods that use HTTP context because
+  // LB will not provide the correct client implementation.
```

Comment:
Methods like `ServerService.getPack` are remote methods defined using the http ctx object. See https://github.com/strongloop/strong-mesh-models/blob/feature/refactor-client/common/models/service.js#L59-59. This means that when you boot up the models, LB tried to fill in the implementation but does it incorrectly. The `client/models/server-service.js` file implements this correctly.

The next problem is load order. `client/models/server-service.js` needs to be loaded after models are booted and attached to the remote datasource, otherwise they will be over-ridden by the incorrect implementation.

I was trying to explain the above in a concise manner.